ethexe/contracts/.env.example

```diff
@@ -3,6 +3,11 @@ SEPOLIA_RPC_URL="https://ethereum-sepolia-rpc.publicnode.com"
 HOLESKY_RPC_URL="https://ethereum-holesky-rpc.publicnode.com"
 PRIVATE_KEY="0x0000000000000000000000000000000000000000000000000000000000000000"
 ETHERSCAN_API_KEY="AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+# example of valid ROUTER_AGGREGATED_PUBLIC_KEY
```

Comment:
uncomment it?

Reply:
I don't want to make it default parameters because whoever is doing deploy should be responsible for public key that can control commits.